language/English/strings.po

```diff
@@ -3321,7 +3321,143 @@ msgctxt "#802"
 msgid "%s of %s available"
 msgstr ""
 
-#empty strings from id 803 to 849
+#. Message in a dialog when a user wants to delete a timer that is part of a repeating schedule
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+#: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp
+msgctxt "#803"
+msgid "Do you also want to delete all other timers from this repeating schedule?"
+msgstr ""
+
+#. Label of spinner control for choosing the timer type (once, weekdays, mondays,..)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#804"
+msgid "Schedule type:"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#805"
+msgid "Once"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#810"
+msgid "Weekdays (Mon-Fri)"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#811"
+msgid "Weekends (Sat-Sun)"
+msgstr ""
+
+#. Type selection when we opened an unsupported timer with 'edit timer'
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#812"
+msgid "Unsupported repeating timer"
```

Comment:
hmm, should we maybe rename this to just "unsupported timer"? I don't think it matters much for which reason it's unsupported.